main.py

```diff
@@ -298,7 +304,7 @@ def main():
                     payload.snr,
                     payload.delta_freq,
                     payload.power,
-                    payload.dist,
+                    int(payload.dist),
```

Comment:
Shouldn't need to cast to int, as the string format takes care of the presentation